### ISSUE TITLE
Taxon navigation isn't working on staging

### ISSUE THREAD

**Issue by bruvellu** [nelas/cifonauta#213]
Clicking on a thumbnail in the taxon navigation isn't working on staging. It's giving an exception in the console:

```
(index):18911 Uncaught ReferenceError: current_title is not defined
    at HTMLAnchorElement.<anonymous> ((index):18911:46)
    at HTMLDocument.dispatch (jquery-1.7.2.min.js:3:4816)
    at HTMLDocument.i (jquery-1.7.2.min.js:3:709)
```

It's probably some div classes/ids that changed with the new layout.

**Comment by bruvellu** [comment]
Probably because of changes in the order of blocks and changes in the `media_page`:

- production: https://github.com/bruvellu/cifonauta/blob/production/meta/templates/media_page.html
- staging: https://github.com/bruvellu/cifonauta/blob/develop/meta/templates/media_page.html

**Comment by bruvellu** [comment]
The order of the blocks was not the main issue, but I fixed it here ad7da616f2f96fb5ad63004d9d9a8bf9a31cdc96. Now the aside is back at the original sidebar block. I had to make small changes to make sure the aside responsiveness works as expected. See here 107f9432cfbd7ecad87d54eaccfb62eb8d814e80. This may need a review.

**Comment by bruvellu** [comment]
The real issue was in a small script in [`media_page_ajax.html`](https://github.com/bruvellu/cifonauta/blame/107f9432cfbd7ecad87d54eaccfb62eb8d814e80/meta/templates/media_page_ajax.html). It only gets the title of the current window and saves it to change the title when the user navigates using the taxon navigation. This is because ajax calls don't refresh the page.

The script was malfunctioning because the quotes around the variable were gone. One of the single quote marks for the variable disappeared with this commit 6c75f0de74d3509b4355a41c881539d5f1f99f12 (see [here](https://github.com/bruvellu/cifonauta/commit/6c75f0de74d3509b4355a41c881539d5f1f99f12?diff=unified#diff-dcdb842c55abc6e312a5a2204217bbf211fd7e54b82e332cb92a9e3585762323L142)). Then the other single quote disappeared with this commit 81cc38204121f52fcf7ad1ac26895a85bad365b3 (see [here](https://github.com/bruvellu/cifonauta/commit/81cc38204121f52fcf7ad1ac26895a85bad365b3#diff-dcdb842c55abc6e312a5a2204217bbf211fd7e54b82e332cb92a9e3585762323L156)).

![Screenshot from 2023-08-18 23-27-42](https://github.com/bruvellu/cifonauta/assets/361179/cd0062a3-b866-426e-92a8-81175e44ac65)

This was probably caused by some automatic linting/formatting.

But it works now!
